abc.templates/infra/spec.yaml

```diff
@@ -21,6 +21,11 @@ inputs:
   - name: 'custom_name'
     desc: 'A custom name for Github Token Minter'
     default: 'github-token-minter'
+    rules:
+      - rule: 'size(custom_name) <= int(19)'
```

Comment:
This is the longest allowable length before reaching the first GCP resource name limitation.